mgo.go

```diff
@@ -499,6 +499,16 @@ func (s *MgoSuite) TearDownSuite(c *gc.C) {
 	mgo.SetLogger(nil)
 }
 
+// MustDial returns a new connection to the MongoDB server, and panics on
+// errors.
+func (inst *MgoInstance) MustDial() *mgo.Session {
+	s, err := mgo.DialWithInfo(inst.DialInfo())
+	if err != nil {
+		panic(err)
```

Comment:
It is generally standard practice for a "MustX" function to panic if it cannot do X. I don't know why we need MustDial, rather than actually handling the error. As this is Testing infrastructure, I suppose it is ok, but I will note that failing an assertion is often *much* easier to debug than a panic.